File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.py
arr = [int(input()) for _ in range(9)]
arr.sort()

ans = []
for i in range(9):
    for j in range(i, 9):
        if sum(arr) - arr[i] - arr[j] == 100:
            ans = [i, j]
            break
    if len(ans) == 2:
        break

for a in arr:
    if a == arr[ans[0]] or a == arr[ans[1]]:
        continue
    else:
        print(a)

File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.java
import java.util.*;
import java.io.*;

public class Main {

	public static void main(String[] args) {
        
		Scanner sc = new Scanner(System.in);

		int sum = 0;
		int[] arr = new int[9];

		for (int i = 0; i < 9; i++) {
			arr[i] = sc.nextInt();
			sum += arr[i];
		}
		
		Arrays.sort(arr);

		for (int i = 0; i < 9; i++) {
			for (int j = i + 1; j < 9; j++) {
				if (sum - arr[i] -arr[j] == 100) {
					for (int h:arr) {
						if (h != arr[i] && h!= arr[j]) {
							System.out.println(h);
						}
					}
					sum = 100;
					break;
				}
			}
			if (sum == 100) {
				break;
			}
		}

	}

}
